src/components/Header/Header.jsx

```diff
@@ -128,19 +138,35 @@ const Header = () => {
 };
 
 const StatusIcon = ({ connector }) => {
-  if (connector === argentX) {
-    return <SvgIcon component={argentXIcon} />;
+  if (connector.options.id === 'argentX') {
```

Comment:
its always a good practice to put checks where we have deep chaining, use optional chaining in this case